### PR TITLE
Improve documentation plots time unit

### LINKED ISSUE #405: Improve plots in documentation
## General

* pyfar version: 0.5.1

## Description

The plots in the documentation that have a time axis could be improved. The time axis is given in seconds in most cases but using milliseconds, samples, etc. would often be better. After building the documentation using `sphinx make html`, the plots are locally stored at **pyfar/docs/_build/plot_directive**

### PR DESCRIPTION
**Pull request by pingelit**
### Which issue(s) are closed by this pull request?

Closes #405

### Changes proposed in this pull request:

Adapt the time unit in the documentation plots so that they are easier to read.

### CODE REVIEW

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
Thanks for the effort. Most plots looks fine now. I found some that could be improved further
- the second example in dsp.normalize looks as if it was plotted with different time units
- the time signal in dsp.filter.reconstructing_fractional_octave_bands could be plotted in ms
- ms could be used for the time and group delay plots in plot.custom_subplots

Might it be that you branched from the master? If yes you could already merge develop in your branch to see if there are any conflicts or things that need updates as well.

File: pyfar/dsp/filter/gammatone.py [hunk 1]
```diff
@@ -97,9 +97,9 @@ class GammatoneBands():
         >>>
         >>> y = GFB.reconstruct(real, imag)
         >>> plt.figure()
-        >>> ax = pf.plot.time_freq(
-        ...     y, label="manipulated and reconstructed and impulse")
-        >>> ax[0].set_xlim(0, .02)
+        >>> ax = pf.plot.time_freq(y, unit='ms',
+        ...     label="manipulated and reconstructed and impulse")
```
Comment (mberz) [inline pyfar/dsp/filter/gammatone.py:101]: indentation level, please check for flake 8 conformity

**Review by mberz — CHANGES_REQUESTED** [review]

File: pyfar/dsp/filter/gammatone.py (hunk 1 above)
Comment (pingelit) [reply pyfar/dsp/filter/gammatone.py:101]: When indenting this, flake8 fails as the line is too long. Should I indent as much as possible?

Comment (f-brinkmann) [reply pyfar/dsp/filter/gammatone.py:101]: In that case, you could use more lines, e.g., by by starting a new line right after `pf.plot.time_freq(`

Comment (pingelit) [reply pyfar/dsp/filter/gammatone.py:101]: And then just indent 4 spaces?

Comment (pingelit) [reply pyfar/dsp/filter/gammatone.py:101]: The function name and the label are too long otherwise

Comment (f-brinkmann) [reply pyfar/dsp/filter/gammatone.py:101]: yes, one tab (4 spaces) are the usual indent if splitting directly after `(`

**Comment by pingelit** [comment]
- Yes, my branch was based on main. I noticed it after I made the PR. `develop` is now merged in this branch.
- I've fixed the indentation issues.
- `dsp.normalize` was missing a unit directive, which is fixed now.
- `dsp.filter.reconstructing_fractional_octave_bands` is now plotted in microseconds. However I cannot see the reconstructed pulse. The plotting code itself looks correct to me. Is this correct?
- `plot.custom_subplots` uses the default parameters for the given plots. I don't think this can be changed.

I also noticed that the plot for `dsp.minimum_phase` is missing the time plot x axis label. Should I fix this in this PR as well?

**Comment by f-brinkmann** [comment]
> I also noticed that the plot for dsp.minimum_phase is missing the time plot x axis label. Should I fix this in this PR as well?

Yes, I think that should be fine :)

**Pull request merged**